adapters/repos/db/shard_shutdown.go

```diff
@@ -22,6 +22,12 @@ import (
 	"github.com/weaviate/weaviate/entities/storagestate"
 )
 
+func (s *Shard) Shutdown(ctx context.Context) (err error) {
+	s.WantShutdown.Store(true)
+	s.performShutdown(ctx)
+	return nil
```

Comment:
should `Shutdown` ever return an error? right now error from `s.performShutdown(ctx)` is ignored, not even logged